iampolicymanagementv1/iam_policy_management_v1.go

```diff
@@ -42,7 +42,7 @@ type IamPolicyManagementV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://iam.cloud.ibm.com"
+const DefaultServiceURL = "https://iam.test.cloud.ibm.com"
```

Comment:
Looks like you were using the "test" version of the API definition to generate code.
We really should be using the version that will be merged into the production branch so that we don't have references to the staging area in the generated code.  This applies to the default service URL here as well as potentially other places where examples from the API definition are used to generate tests, etc.